userspace/libscap/engine/noop/noop.h

```diff
@@ -19,6 +19,7 @@ limitations under the License.
 
 #include "engine_handle.h"
 #include "scap_vtable.h"
+#include "scap.h"
 
```

Comment:
We don't need to include the whole kitchen sink here, do we?

```suggestion

struct scap_stats_v2;
```

(and maybe an `#include <stdint.h>` if it isn't already here somewhere)